vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -27,5 +27,6 @@ namespace Microsoft.ReactNative {
     [noexcept2] String DebugHost { get; set; };
     [noexcept2] String DebugBundlePath { get; set; };
     [noexcept2] String BundleRootPath { get; set; };
+    [noexcept2] UInt16 DebuggerPort { get; set; };
```

Comment:
>[noexcept2] [](start = 4, length = 11)

Please note that after we upgrade to the new C++/WinRT version 2.0.200316.3 in PR #4449 we will not be able to use this attribute for some time. Depending which PR submitted first, one of us must remove it here.
Could you remove it in case if you plan to commit another change before PR?

Reply:
Thanks for the heads up!
Will remove it in next commit before merging.